examples-extra/docs_example/package.json

```diff
@@ -26,8 +26,8 @@
     "@osdk/cli.cmd.typescript": "workspace:^",
     "@types/react": "^18.2.79",
     "@types/react-dom": "^18.2.25",
-    "@typescript-eslint/eslint-plugin": "^6.10.0",
-    "@typescript-eslint/parser": "^6.10.0",
+    "@typescript-eslint/eslint-plugin": "^7.8.0",
+    "@typescript-eslint/parser": "^7.8.0",
     "@vitejs/plugin-react": "^4.2.1",
     "autoprefixer": "^10.4.19",
     "eslint": "^8.57.0",
```

Comment:
Note that there are still peer dependencies issues with eslint 8 specified here and eslint 9 used within this monorepo

```
 WARN  Issues with peer dependencies found
.
├─┬ @typescript-eslint/parser 7.8.0
│ └── ✕ unmet peer eslint@^8.56.0: found 9.1.1
├─┬ eslint-plugin-import 2.29.1
│ └── ✕ unmet peer eslint@"^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8": found 9.1.1
├─┬ typescript-eslint 7.7.1
│ ├── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│ ├─┬ @typescript-eslint/eslint-plugin 7.7.1
│ │ ├── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│ │ └─┬ @typescript-eslint/type-utils 7.7.1
│ │   ├── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│ │   └─┬ @typescript-eslint/utils 7.7.1
│ │     └── ✕ unmet peer eslint@^8.56.0: found 9.1.1
│ └─┬ @typescript-eslint/parser 7.7.1
│   └── ✕ unmet peer eslint@^8.56.0: found 9.1.1
└─┬ eslint-plugin-unused-imports 3.1.0
  └── ✕ unmet peer eslint@8: found 9.1.1
```

However, that's already the case on main before this PR and for these rendered examples with `ESLINT_USE_FLAT_CONFIG=false` I think things are still working fine. We need the plugins to officially support eslint 9 before we can fully move the templates over as well